modules/core/src/main/java/org/apache/ignite/internal/util/GridLogThrottle.java

```diff
@@ -44,13 +49,39 @@
     private static final ConcurrentMap<IgniteBiTuple<Class<? extends Throwable>, String>, Long> errors =
         new ConcurrentHashMap8<>();
 
+    /** Scheduler. */
+    private static final ScheduledExecutorService scheduler = Executors.newScheduledThreadPool(1);
+
+    /** Future instance to stop Scheduler. */
```

Comment:
I think this javadoc is useless and wrong.
We *manage*(not stopping) scheduled *task*, not scheduler itself.